core/src/main/java/io/cucumber/core/backend/TestCaseState.java

```diff
@@ -1,9 +1,11 @@
 package io.cucumber.core.backend;
 
+import io.cucumber.plugin.event.TestStep;
```

Comment:
The `backend` api and the `plugin` and `java` api can evolve at different speeds. The `plugin` API is implemented by third parties that are slow to change while the `backend` almost entirely implemented by projects that are part of Cucumber. As such neither `backend` nor `java` should not depend or use any classes from `plugin`.